calendar.php

```diff
@@ -2232,7 +2232,8 @@
     $in_month = 0;
 
     // Iterate weeks (each week gets a row)
-	$week = $day_bit = '';
+    $week = [];
```

Comment:
Needs to be `array()` since we maintian PHP 5.2 compatibility in 1.8 unfortunately.

Reply:
1.8? but it's dev1.9 branch

Reply:
I will check it tomorrow, but I'm pretty sure that on 1.8 that bug is not present.